src/components/structures/RoomView.tsx

```diff
@@ -354,6 +355,20 @@ export class RoomView extends React.Component<IRoomProps, IRoomState> {
             : MainSplitContentType.Timeline;
     };
 
+    private checkRightPanel = (room) => {
+        // This is a hack to hide the chat. This should not be necassary once the right panel
+        // phase is stored per room. (need to be done after check widget so that mainSplitContentType is updated)
+        if (RightPanelStore.getSharedInstance().roomPanelPhase === RightPanelPhases.Timeline
+            && this.state.showRightPanel
+            && !WidgetLayoutStore.instance.hasMaximisedWidget(this.state.room)) {
```

Comment:
```suggestion
        if (
            RightPanelStore.getSharedInstance().roomPanelPhase === RightPanelPhases.Timeline &&
            this.state.showRightPanel &&
            !WidgetLayoutStore.instance.hasMaximisedWidget(this.state.room)
        ) {
```

Reply:
Oh I forgot this one... you commented that earlier already.